README.md

```diff
@@ -101,6 +101,12 @@ Some example modules that can be excluded are:
 - **exports/global**: Exclude the attachment of global jQuery variables ($ and jQuery) to the window.
 - **exports/amd**: Exclude the AMD definition.
 
+As a special case, you may also replace the full jQuery `selector` module by using a special flag `grunt custom:-selector`.
```

Comment:
Is this true, or is it necessary to use `-selector-full`?

Reply:
That was an omission on my part. It doesn't work right now, it will actually print a warning. But I'll update the PR since we now want to use this instead of `custom:-selector-full`.

Reply:
Done. The renaming is done in the latest commit.